packages/quick_actions/quick_actions/android/src/main/java/io/flutter/plugins/quickactions/QuickActionsPlugin.java

```diff
@@ -43,6 +47,10 @@ public void onDetachedFromEngine(FlutterPluginBinding binding) {
   @Override
   public void onAttachedToActivity(ActivityPluginBinding binding) {
     handler.setActivity(binding.getActivity());
+    binding.addOnNewIntentListener(this);
+    if (isInitialized) {
```

Comment:
could we add comments to the code? That will help the next maintainer

Reply:
Done.